request.go

```diff
@@ -10,6 +10,11 @@ import (
 	"time"
 )
 
+const (
+	requestCheckTimeout            = 2 * time.Minute
+	requestServerErrorRetryTimeout = 1 * time.Minute
```

Comment:
It would be great if this can be passed as parameter in the client's configuration.

Reply:
OK, I'll implement it.

Reply:
@itakouna just `requestServerErrorRetryTimeout `, isn't it?